Lorem Ipsum XR Translator/Assets/Scripts/Behaviors/CaptionController.cs

```diff
@@ -0,0 +1,44 @@
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+
+public class CaptionController : MonoBehaviour
+{
+    public GameObject CaptionPrefab;
+
+    List<GameObject> CaptionList;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        CaptionList = new List<GameObject>();    
+    }
+
+    /// <summary>
+    /// Deletes all current captions in world and clears the list
+    /// </summary>
+    public void ClearCaptions()
+    {
+        // Pooling our caption objects might be more performant?
+        foreach (var c in CaptionList)
+        {
+            Destroy(c);
+        }
+        CaptionList.Clear();
+    }
+
+    /// <summary>
+    /// Creates a new caption object and places it in the world facing towards the player
+    /// </summary>
+    /// <param name="captionName">The text to be displayed on the caption</param>
+    /// <param name="captionLocation">The Vector location of where the caption should be placed</param>
+    public void CreateCaption(string captionName, Vector3 captionLocation)
+    {
+        GameObject caption = Instantiate(CaptionPrefab, captionLocation, 
+            Quaternion.LookRotation(captionLocation - CameraCache.Main.transform.position));
+        caption.transform.Find("CaptionText").GetComponent<TextMeshPro>().text = captionName;
+
+        CaptionList.Add(caption);
```

Comment:
In my opinion, I think it'd be more flexible if you can pass rotation information to this method. Is there any reason behind this implementation?